docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
+FROM ubuntu:latest
```

Comment:
Orion docker based on CentOS 6 in https://github.com/telefonicaid/fiware-orion/pull/1059